plugins/services/src/js/reducers/__tests__/JSONMultiContainer-test.js

```diff
@@ -0,0 +1,84 @@
+const Batch = require("#SRC/js/structs/Batch");
+const { combineParsers } = require("#SRC/js/utils/ParserUtil");
+const { combineReducers } = require("#SRC/js/utils/ReducerUtil");
+const JSONMultiContainerParser = require("../JSONMultiContainerParser");
+const JSONMultiContainerReducers = require("../JSONMultiContainerReducers");
+
+describe("JSONMultiContainer", function() {
+  describe("integrity", function() {
+    it("parse and reduce doesn't change JSON", function() {
```

Comment:
Please adjust the test description to be inline with the rest of out test cases... For Example: "it **should** parse and reduce without changing the JSON". Not a blocker though...